internal/converters/converters.go

```diff
@@ -1,11 +1,11 @@
-package equinix
+package converters
 
 import (
 	"strconv"
 	"strings"
 )
 
-func contains(s []string, e string) bool {
+func Contains(s []string, e string) bool {
```

Comment:
If we build on go 1.21 instead of 1.20 we could replace this with usage of the `slices` standard package (even if we don't upgrade, we could use `x/exp/slices`.

Reply:
Replacement of this function could be saved for a later change, but if we opt to keep this implementation of `Contains`, I think it would fit better in a package named `slices`, since it isn't converting anything.